src/Components/Aspire.Azure.Storage.Blobs/AzureStorageBlobsSettings.cs

```diff
@@ -9,7 +9,7 @@ namespace Aspire.Azure.Storage.Blobs;
 /// <summary>
 /// Provides the client configuration settings for connecting to Azure Blob Storage.
 /// </summary>
-public sealed class AzureStorageBlobsSettings : IConnectionStringSettings
+public class AzureStorageBlobsSettings : IConnectionStringSettings
```

Comment:
Unsealed to allow inheritance in `AzureBlobStorageContainerSettings`. Any thoughts on this? 
I can duplicate the members in there and retain `sealed`...

Reply:
We have unsealed the settings class in other places as well. So I think this is fine.